trace-to-events/src/main.rs

```diff
@@ -38,9 +43,18 @@ struct Cli {
     #[clap(long)]
     event_topic: String,
 
-    #[clap(long, default_value = "127.0.0.1:9090")]
+    /// Determines whether events should register as positive or negative voltage
```

Comment:
Could we avoid using the term "voltage" in favour of "intensity".

Any value in a trace is seen as an arbitrary intensity rather than a voltage given that no ADC scaling is applied.